conf/layer.conf

```diff
@@ -8,6 +8,6 @@ BBFILE_COLLECTIONS += "synopsys-layer"
 BBFILE_PATTERN_synopsys-layer = "^${LAYERDIR}/"
 BBFILE_PRIORITY_synopsys-layer = "6"
 
-LAYERSERIES_COMPAT_synopsys-layer = " honister"
+LAYERSERIES_COMPAT_synopsys-layer = " kirkstone"
```

Comment:
Cannot we just add `kirkstone` along with `honister`? I.e. is there something which makes our layer incompatible with `honister`?
For the record, `kirkstone` is going to be the next LTS - so it's good to rely on it looking forward.